File: OCExample/OCExample/ViewController.m
//
//  ViewController.m
//  OCExample
//
//  Created by 神崎H亚里亚 on 2019/11/27.
//  Copyright © 2019 moxcomic. All rights reserved.
//

#import "ViewController.h"
#import "OCExample-Swift.h"
#import <AriaSwiftyChain/AriaSwiftyChain-Swift.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    [[UIButton alloc] init]
    .setTitleForState(@"按钮", UIControlStateNormal)
    .setTitleColorForState(UIColor.redColor, UIControlStateNormal)
    .setTap(^{
        NSLog(@"被点击了...");
        [UIAlertController showWithTitle:@"" message:nil confirmText:@"OK" handler:nil];
    })
    .addToSuperview(self.view)
    .setFrameWithXYWH(100, 100, 50, 50);
    
    [[UILabel alloc] init]
    .setText(@"文本");
    
    self.topViewController;
}


@end

File: Example/Example/ViewController.swift
//
//  ViewController.swift
//  Example
//
//  Created by 神崎H亚里亚 on 2019/11/27.
//  Copyright © 2019 moxcomic. All rights reserved.
//

import UIKit
import AriaSwiftyChain

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        let btn = UIButton()
            .title("测试", for: .normal, .highlighted)
            .titleColor(.white, for: .normal, .highlighted)
            .backgroundColor(.red)
            .addToSuperview(view)
            .tap({
                print("被点击了")
                UIAlertController.show(withTitle: "被点击了")
            }).makeConstraints({ (make) in
                make.center.equalToSuperview()
                make.width.height.equalTo(50)
            })
        
        UILabel()
            .text("文本")
            .textColor(.black)
            .font(.systemFont(ofSize: 16))
            .addToSuperview(view)
            .makeConstraints({ (make) in
                make.top.equalTo(btn.snp.bottom).offset(10)
                make.centerX.equalTo(btn)
            })
        
        
    }
}

